core/arch/arm/pta/pta_invoke_test.c

```diff
@@ -36,9 +36,9 @@
 
 #include "core_self_tests.h"
 
-#define TA_NAME		"sta_self_tests.ta"
+#define TA_NAME		"invoke_test.pta"
```

Comment:
Minor: why not keep the pural? (`invoke_tests.pta`) for consistency with other tests, see `sub.mk`.